.rubocop.yml

```diff
@@ -122,7 +122,6 @@ Style/RequireOrder:
   Enabled: true
 
 RSpec/SpecFilePathFormat:
-  Enabled: true
```

Comment:
Why?

Reply:
Because this cop is enabled by default (changed from `pending` in #1902).